### PR TITLE
fix: moved community section to different page

### PR DESCRIPTION
**Pull request by harmeetsingh11**
## Related Issue

Closes: #540 

## Description of Changes

Moved Grabtern section to a different page called community.

### Files modified
- `components/layout/Header.js`
- `pages/index.page.js`

### Files added
- `components/Community/CommunityCard.jsx`
- `pages/community.page.js`

## Screenshots
[GrabTern-Grab-Your-Internship.webm](https://github.com/anmode/grabtern-frontend/assets/130365071/46f33f3c-b78b-492f-b83f-0d62a4c0aa60)

## Checklist:

- [x] My code adheres to the established style guidelines of this project.
- [x] I have conducted a self-review of my code.
- [x] My changes have not introduced any new warnings.

@anmode please review this PR.
Thanks

### CODE REVIEW

File: components/Community/CommunityCard.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { ProfileCard, IconLink } from "../UI";
```
Comment (Isha988) [inline components/Community/CommunityCard.jsx:2]: @harmeetsingh11 Remove unused imports `IconLink`, `RiLinkedinLine`, `RiTwitterLine`

File: pages/index.page.js [hunk 2]
```diff
@@ -219,32 +217,6 @@ export default function Home() {
...
-        <Section
-          kicker="Grabtern team"
-          heading="Our Community"
-          subheading="Meet the members of Grabtern Community"
-          align="center"
-        >
-          <div>
-            {carousel === true ? (
-              <OwlCarousel
-                {...teamsOptions}
```
Comment (Isha988) [inline pages/index.page.js:233]: Also remove this` teamsOptions `object from `index.page.js`

**Review by Isha988 — CHANGES_REQUESTED** [review]

**Review by Isha988 — APPROVED** [review]
Looks good to me 👍

**Comment by harmeetsingh11** [comment]
@anmode thanks for merging the PR. Can you please add `level` label to it?

**Comment by anmode** [comment]
Sorrry! My bad added now

**Pull request merged**